### ISSUE TITLE
Create/update recovery code

### ISSUE THREAD

**Issue by anikarni** [pixelated/pixelated-user-agent#924]
When the user submits the back-up email, it should persist the recovery code. This story is to save a static recovery code in Bonafide. However, after this story is done, we still won't have the recovery code accessible to us or have access to the secret. The next stories will take care of that.

## Acceptance Criteria

When I submit my back-up email, a *static* recovery code should be saved for me in Bonafide. If a recovery code already exists, it should be updated.

## In Scope

* Call the /users endpoint to add/update a second password in Bonafide that will be the recovery code  [requires PR]

## Out of Scope

* Generating new recovery code
* Sending the recovery code to the user.
* Encrypting Soledad secret.

## Depends on

#922

**Comment by anikarni** [comment]
This should only happen when the user enters the back-up email, so this should be blocked by #922 :( My mistake
